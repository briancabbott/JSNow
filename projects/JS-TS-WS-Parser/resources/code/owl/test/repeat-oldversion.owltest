x x x x x
---#using owl.v3
x = 'x'{3+}
