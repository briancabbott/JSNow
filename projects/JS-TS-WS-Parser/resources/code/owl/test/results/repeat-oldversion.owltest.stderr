error: explicit repetition is unsupported in versions before owl.v4

  #using owl.v3
         ~~~~~~
  x = 'x'{3+}
      ~~~~~~~

